### PR TITLE
Sanitizing file names in example_bash_decorator DAG

### PR DESCRIPTION
**Pull request by amoghrajesh**
Adding some validation to an example dag

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/example_dags/example_bash_decorator.py
Comment (uranusjr) [inline airflow/example_dags/example_bash_decorator.py]:
```suggestion
        # Sanitizing the file names
```

By the way, Python 3.8 added `shlex.join` that makes this even easier

```python
shlex.join([stat, *files])
```

Comment (amoghrajesh) [reply airflow/example_dags/example_bash_decorator.py]: Oh, didn't know that! Thanks!

Comment (amoghrajesh) [reply airflow/example_dags/example_bash_decorator.py]: Yes, works as expected. Pushed a change

**Review by potiuk — APPROVED** [review]

**Comment by amoghrajesh** [comment]
I will wait on @uranusjr's approval here

**Review by enchant3dmango — APPROVED** [review]

**Comment by amoghrajesh** [comment]
Hey @uranusjr are you good with this PR?

**Comment by potiuk** [comment]
I think the comment was addressed. Let's just merge it.

**Pull request merged**
